### ISSUE TITLE
A11y_VSCode_VersionControl_Screenreader_Voice Over does not announce information about operation.

### ISSUE THREAD

**Issue by prasadaps** [microsoft/vscode#77065]
Environment Details:
VSCode Version : 1.36.0
OS Version :  10.14.5
Additional Details:
MAS Violated : MAS 4.1.3

GitHubTags: #A11yMAS;   #A11yTCS; #Win32; #DesktopApp; #A11y_VSCodeEditor; #MAS4.1.3

Repro Steps:
1.Open Visual studio code editor
2.Using F1 key navigate to the palette, type "git branch" and select "Create Branch"
3. Enter the branch name in the edit field and hit enter. Observe the change in branch name at the bottom left corner
4. Publish branch to the remote Git repository by navigating to the bottom left pane, arriving at the cloud button next to the git branch and hit ENTER
5.Start Screen reader.
6. Observe screen reader announcement when click on publish button.
Actual:
when user click on Commit all option or on the undo last commit option or on the publish button the screen reader does not announce status message of performed action i.e. committing action undoing the last action or publishing the to git.
Expected:
screen reader should announce status message of whatever operation is performing in background
Recommendations:
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx
User Impact:
Screen reader user will not get informed with the action performed.
MAS Reference:
MAS 4.1.3- https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7B684573A7-B089-4131-9B39-0009054125D3%7D&file=MAS%204.1.3%20%E2%80%93%20Status%20Messages.docx&action=default&mobileredirect=true&cid=1f363df4-1eaa-4680-9e0a-c63b09081427
Attachment For Reference:
[Mac_A11y_VSCode_VersionControl_ScreenReader_ VoiceOver does not announce anything abouct background operation.pptx](https://github.com/microsoft/vscode/files/3376437/Mac_A11y_VSCode_VersionControl_ScreenReader_.VoiceOver.does.not.announce.anything.abouct.background.operation.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
@joaomoreno it is correct that you are not annoucning anything for git operations.
Leaving this open and assigning to you so you decide if you would like to announce something once a git opertion is completed. Might make sense. Also not sure if you already have an issue for this

**Comment by joaomoreno** [comment]
> Leaving this open and assigning to you so you decide if you would like to announce something once a git opertion is completed. Might make sense. Also not sure if you already have an issue for this

@isidorn Can't do it unless we have extension API for announcing accessibility stuff. So I'll let you decided whether we want to have that.

**Comment by georgebatalinski** [comment]
@isidorn any insight into this? what is the `extension API for announcing accessibility stuff`?

**Comment by isidorn** [comment]
@georgebatalinski We do not have the extension API for accessibility.
@joaomoreno are there other areas in Git in which you would profit from us adding such an API.
If we find more use cases from extension authers I would be more willing to push this.

**Comment by joaomoreno** [comment]
The same argument posed by @prasadaps can pretty much be applied to any contributed command:

- Variable `foo` has been extracted
- Azure Storage Account has been created
- etc.

Though I'm not entirely convinced this will be a good thing.

**Comment by isidorn** [comment]
Let's leave this one open for now until we get more user feedback. Also not convinced this is a good suggestion.

**Comment by joaomoreno** [comment]
ping @isidorn
